apps/meteor/ee/app/livechat-enterprise/server/api/outbound.ts

```diff
@@ -1,55 +1,70 @@
 import type { IOutboundProvider } from '@rocket.chat/core-typings';
-import { ajv } from '@rocket.chat/rest-typings';
+import { ajv } from '@rocket.chat/rest-typings/src/v1/Ajv';
 
 import { API } from '../../../../../app/api/server';
-import { isGETOutboundProviderParams } from '../outboundcomms/rest';
+import { isGETOutboundProviderParams, isOutboundProviderMetadataSchema } from '../outboundcomms/rest';
 import { outboundMessageProvider } from './lib/outbound';
 import type { ExtractRoutesFromAPI } from '../../../../../app/api/server/ApiClass';
 
-const outboundCommsEndpoints = API.v1.get(
-	'omnichannel/outbound/providers',
-	{
-		response: {
-			200: ajv.compile<{ providers: IOutboundProvider[] }>({
-				type: 'object',
-				properties: {
-					providers: {
-						type: 'array',
-						items: {
-							type: 'object',
-							properties: {
-								providerId: {
-									type: 'string',
-								},
-								providerName: {
-									type: 'string',
-								},
-								supportsTemplates: {
-									type: 'boolean',
-								},
-								providerType: {
-									type: 'string',
-								},
-								documentationUrl: {
-									type: 'string',
+const outboundCommsEndpoints = API.v1
+	.get(
+		'omnichannel/outbound/providers',
+		{
+			response: {
+				200: ajv.compile<{ providers: IOutboundProvider[] }>({
+					type: 'object',
+					properties: {
+						providers: {
+							type: 'array',
+							items: {
+								type: 'object',
+								properties: {
+									providerId: {
+										type: 'string',
+									},
+									providerName: {
+										type: 'string',
+									},
+									supportsTemplates: {
+										type: 'boolean',
+									},
+									providerType: {
+										type: 'string',
+									},
 								},
 							},
 						},
 					},
-				},
-			}),
+				}),
+			},
+			query: isGETOutboundProviderParams,
+			authRequired: true,
 		},
-		query: isGETOutboundProviderParams,
-		authRequired: true,
-	},
-	async function action() {
-		const { type } = this.queryParams;
+		async function action() {
+			const { type } = this.queryParams;
 
-		const providers = outboundMessageProvider.listOutboundProviders(type);
-		return API.v1.success({
-			providers,
-		});
-	},
-);
+			const providers = outboundMessageProvider.listOutboundProviders(type);
+			return API.v1.success({
+				providers,
+			});
+		},
+	)
+	.get(
+		'omnichannel/outbound/providers/:id/metadata',
+		{
+			response: {
+				200: isOutboundProviderMetadataSchema,
```

Comment:
The endpoint could return a 400 response, in those cases, this will fail as there's no definition for a 400. Can you add one?